documents/affiliated_package_review_process.md

```diff
@@ -0,0 +1,172 @@
+ # Procedure for the proposal, review, and acceptance of Astropy-affiliated packages
+
+This document describes the procedure for proposing, reviewing, and making a
+decision about Astropy-affiliated packages. This is meant as a record of the
+procedure and not the first place people should go to for - for instructions
+on proposing an affiliated package, see **Becoming an Affiliated Package** on
+[this](http://www.astropy.org/affiliated/index.html) page.
+
+## Proposing an affiliated package
+
+*Note that this section is written as if to the affiliated package author*
```

Comment:
"as if to the" -> "to the" or "this section is meant for affiliated package authors"?